src/components/atoms/Typography/Typography.tsx

```diff
@@ -0,0 +1,39 @@
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+export interface ItypographyProps {
```

Comment:
you can extend TypographyProps from mui so that you need not to redefine variant